src/main/resources/api.yaml

```diff
@@ -93,7 +93,8 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  /accounts/@me:
+
+  /accounts/@update:
```

Comment:
That's not restful. "PUT" has expressed that the API is for update. Don't put operation on URL.
I think your problem can be fixed by simply changing `@me` to `{id}`

Reply:
Thx for pointing out. Indeed @update sounds wired and expose the operation. Now it has been fixed. I change it back to the original design accounts/@me (it was the `$curl` problem in the previous test leading to the `conflict in domain` problem).